content/docs/for-buildpack-authors/how-to/write-buildpacks/create-slice-layers.md

```diff
@@ -3,8 +3,34 @@ title="Create slice layers"
 weight=99
 +++
 
+A `Slice` represents a layer in the `<app>` directory to be exported during the `export` phase.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+A buildpack may specify sub-paths within `<app>` as `slices` in `launch.toml`. The lifecycle will create separate layers during the [export](https://buildpacks.io/docs/for-buildpack-authors/concepts/lifecycle-phases/#phase-5-export) phase for each slice with one or more files or directories.
+
+## Key Points
```

Comment:
I think this section would make more sense after Implementation Steps

Reply:
re-ordered